identity-model/test/IdentityModel.Tests/HttpClientExtensions/UserInfoExtensionsTests.cs

```diff
@@ -161,7 +161,7 @@ public async Task Non_json_response_should_set_raw()
         response.IsError.ShouldBeFalse();
         response.ErrorType.ShouldBe(ResponseErrorType.None);
         response.HttpStatusCode.ShouldBe(HttpStatusCode.OK);
-        response.Claims.ShouldBeNull();
+        response.Claims.ShouldBeEmpty();
```

Comment:
I think we're okay with this behavior change - it was the only test that broke when I simplified the condition of when we invoke `ProtocolResponse.IntiailizeAsync`